Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/openSAML/OpenSAML2ComponentBuilder.java

```diff
@@ -123,124 +104,58 @@
     private static final String DEFAULT_ISSUER_VALUE = "CN=SAML User,OU=SU,O=SAML User,L=Los Angeles,ST=CA,C=US";
 
     /**
-     * The assertion builder.
-     */
-    //private final SAMLObjectBuilder<Assertion> assertionBuilder;
-
-    /**
-     * The name id builder.
-     */
-    //private final SAMLObjectBuilder<NameID> nameIdBuilder;
-
-    /**
-     * The conditions builder.
-     */
-    //private final SAMLObjectBuilder<Conditions> conditionsBuilder;
-
-    /**
-     * The action element builder.
-     */
-    //private final SAMLObjectBuilder<Action> actionElementBuilder;
-
-    /**
-     * The authorization decision statement builder.
+     * The evidence builder.
      */
-    // private final SAMLObjectBuilder<AuthzDecisionStatement> authorizationDecisionStatementBuilder;
+    private final SAMLObjectBuilder<Evidence> evidenceBuilder;
 
     /**
-     * The string builder.
+     * The xsAnyBuilder.
      */
-    //private final XSStringBuilder stringBuilder;
+    private final XSAnyBuilder xsAnyBuilder;
 
     /**
-     * The evidence builder.
+     * The instance.
      */
-    private final SAMLObjectBuilder<Evidence> evidenceBuilder;
-
-    private final XSAnyBuilder xsAnyBuilder;
+    private static OpenSAML2ComponentBuilder iNSTANCE;
```

Comment:
This doesn't follow any naming convention within the CONNECT coding standards.  As a private static, it would be "instance"; as a private static final (with a single point of assignment), it would be "INSTANCE".